From edf81b113f5c3a3db8c2a60c1fccb898fa2fbd24 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 18 Feb 2020 12:12:41 +0100
Subject: [PATCH 1226/1852] dt-bindings: xilinx: Document new evaluation boards

commit 81db5bc9bbe3fa41d97338eaca49e1ffbd440af0 from
https://github.com/Xilinx/linux-xlnx.git

Just wire new boards to pass checking done by make dtbs_check

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/arm/xilinx.yaml       | 42 +++++++++++++++----
 1 file changed, 35 insertions(+), 7 deletions(-)

diff --git a/Documentation/devicetree/bindings/arm/xilinx.yaml b/Documentation/devicetree/bindings/arm/xilinx.yaml
index c73b1f5c7f49..265020976bfd 100644
--- a/Documentation/devicetree/bindings/arm/xilinx.yaml
+++ b/Documentation/devicetree/bindings/arm/xilinx.yaml
@@ -58,12 +58,6 @@ properties:
           - const: xlnx,zynqmp-zc1254
           - const: xlnx,zynqmp
 
-      - description: Xilinx internal board zc1275
-        items:
-          - const: xlnx,zynqmp-zc1275-revA
-          - const: xlnx,zynqmp-zc1275
-          - const: xlnx,zynqmp
-
       - description: Xilinx 96boards compatible board zcu100
         items:
           - const: xlnx,zynqmp-zcu100-revC
@@ -91,6 +85,7 @@ properties:
         items:
           - enum:
               - xlnx,zynqmp-zcu104-revA
+              - xlnx,zynqmp-zcu104-revC
               - xlnx,zynqmp-zcu104-rev1.0
           - const: xlnx,zynqmp-zcu104
           - const: xlnx,zynqmp
@@ -107,8 +102,41 @@ properties:
         items:
           - enum:
               - xlnx,zynqmp-zcu111-revA
-              - xlnx,zynqmp-zcu11-rev1.0
+              - xlnx,zynqmp-zcu111-rev1.0
           - const: xlnx,zynqmp-zcu111
           - const: xlnx,zynqmp
 
+      - description: Xilinx evaluation board zcu208
+        items:
+          - enum:
+              - xlnx,zynqmp-zcu208-revA
+              - xlnx,zynqmp-zcu208-rev1.0
+          - const: xlnx,zynqmp-zcu208
+          - const: xlnx,zynqmp
+
+      - description: Xilinx evaluation board zcu216
+        items:
+          - enum:
+              - xlnx,zynqmp-zcu216-revA
+              - xlnx,zynqmp-zcu216-rev1.0
+          - const: xlnx,zynqmp-zcu216
+          - const: xlnx,zynqmp
+
+      - description: Xilinx evaluation board zcu1275
+        items:
+          - enum:
+              - xlnx,zynqmp-zcu1275-revA
+              - xlnx,zynqmp-zcu1275-revB
+              - xlnx,zynqmp-zcu1275-rev1.0
+          - const: xlnx,zynqmp-zcu1275
+          - const: xlnx,zynqmp
+
+      - description: Xilinx evaluation board zcu1285
+        items:
+          - enum:
+              - xlnx,zynqmp-zcu1285-revA
+              - xlnx,zynqmp-zcu1285-rev1.0
+          - const: xlnx,zynqmp-zcu1285
+          - const: xlnx,zynqmp
+
 ...
-- 
2.31.1

